gradle/libs.versions.toml

```diff
@@ -0,0 +1,156 @@
+[versions]
+netty = "4.1.108.Final"
+slf4j = "2.0.13"
+dockerapi = "3.3.6"
+guava = "32.0.1-jre"
+protobuf = "3.25.5"
+httpclient5 = "5.2.1"
+aws-msk-iam-auth = "2.0.3"
+log4j = "2.23.1"
+lombok = "1.18.30"
+kafka-clients = "3.6.0"
+jmeter = "5.6.3"
+jcommander = "1.85"
+jackson = "2.16.2"
+jupiter = "5.10.2"
+jmespath-core = "0.6.0"
+commons-compress = "1.26.0"
+awssdk = "2.25.19"
+awscrt = "0.29.18"
+jolt-core = "0.1.7"
+opentelemetry = "1.34.1"
+opentelemetry-semconv = "1.23.1-alpha"
+disruptor = "4.0.0"
+bouncycastle = "1.78"
+testcontainers = "1.19.7"
+opensearch-testcontainers = "2.0.1"
+mockito = "5.11.0"
+toxiproxy = "2.1.7"
+hamcrest = "2.2"
+jmh-core = "1.36"
+semver4j = "5.3.0"
+json = "20240303"
+graalvm = "24.1.1"
+reactor-test = "3.7.3"
+reactor-netty = "1.1.18"
+lucene7 = '7.7.3'
+lucene9 = "9.11.1"
+jni-java = "2.7.3"
+errorprone = "2.32.0"
+commons-collections = "4.4"
+jenkins-pipeline-unit = "1.23"
+groovy = "3.0.23"
```

Comment:
can we make these alphabetical to make it easier to see dups/conflicts?

Reply:
Yup - I'll sort and then make edits there are a couple names such as 'aws' that I figure we should better name and group